sos/report/plugins/aap_controller.py

```diff
@@ -59,6 +60,13 @@ def setup(self):
             "umask -p",
         ])
 
+        # run_wsbroadcast is replaced with run_wsrelay in AAP 2.4 and above
+        awx_ver = self.collect_cmd_output('awx-manage --version')
+        if sos_parse_version(awx_ver['output']) > sos_parse_version('4.4.99'):
+            self.add_cmd_output("awx-manage run_wsrelay --status")
+        else:
+            self.add_cmd_output("awx-manage run_wsbroadcast --status")
```

Comment:
Put this under `if awx_ver['status'] == 0:` test. Then the CI tests will pass :) - if one (accidentally) runs this plugin on a system with no `awx-manage` binary, `awx_ver` gets `{'status': 127, 'output': b'', 'truncated': False}` and the binary empty string causes an uncaught exception.

@TurboTurtle , wrt the `b''` output, does it make sense to modify https://github.com/sosreport/sos/blob/main/sos/utilities.py#L367:

    -    stdout = b""
    +    stdout = ""

to prevent such errors? We should guard such situations via the "status is zero?" test, but still..?